tools/src/merger/OpenApiMerger.ts

```diff
@@ -9,11 +9,13 @@ import GlobalParamsGenerator from './GlobalParamsGenerator'
 export default class OpenApiMerger {
   root_folder: string
   spec: Record<string, any>
+  silent: boolean
 
   paths: Record<string, Record<string, OpenAPIV3.PathItemObject>> = {} // namespace -> path -> path_item_object
   schemas: Record<string, Record<string, OpenAPIV3.SchemaObject>> = {} // category -> schema -> schema_object
 
-  constructor (root_folder: string) {
+  constructor (root_folder: string, silent: boolean = false) {
+    this.silent = silent
```

Comment:
Passing `silent` around is going to get messy. Follow the logger pattern in most applications - make a logger class and pass that around (ideally assign it in the constructor). The logger should have an `info`, `warn` and `error` method that checks log level. Then you can implement things like `--verbose` or `--silent` and the way output is rendered without having to change everything in the code.